activation/atxdb.go

```diff
@@ -785,15 +797,15 @@ func (db *DB) HandleAtxData(data []byte, syncer service.Fetcher) error {
 
 // FetchAtxReferences fetches positioning and prev atxs from peers if they are not found in db
 func (db *DB) FetchAtxReferences(atx *types.ActivationTx, f service.Fetcher) error {
-	if atx.PositioningATX != *types.EmptyATXID {
+	if atx.PositioningATX != *types.EmptyATXID && atx.PositioningATX != db.goldenATXID {
```

Comment:
do you need the first condition now? should there be a case where an ATX is empty?

Reply:
Just an extra check. However, it's actually redundant. Removed.